app/nginx/sites-enabled/preview.coucher.org

```diff
@@ -0,0 +1,35 @@
+server {
+    # we want to be able to preview frontend PRs, etc, so they're being uploaded to e.g.
+    # https://couchers-dev-assets.s3.amazonaws.com/frontend/ci-test-results-90b9e9ee/index.html
+    # the problem is that this tries to load js from /js/..., which gets screwed up being in a subfolder
```

Comment:
So fix the frontend to load from `js/...`, i.e. relative path.

Reply:
That's the long-term solution, I'll open an issue. OK for now?

Reply:
Sure, I just enjoy complaining :D

Reply:
Ahahah, that's alright.

Actually, I think we can't do relative paths, because the same data is sometimes loaded from different paths. i.e. you actually get the same HTML for both https://app.couchers.org/user/aapeli and https://app.couchers.org/, due to this whole HTML5 history navigation weirdness...

Reply:
I've never gotten the hook of "Single-page application", I'm back in the CGI days still... :older_adult: